### DISCUSSION TITLE
Fprime support for Highspeed USB3.0 interface

### DISCUSSION THREAD

**Discussion by satzguyz** [nasa/fprime#1399] [Q&A]
Hello,

Is there any support for high-speed interfaces like USB 3.1 in FPrime SW? Any future milestones to add support for the same ?

**Comment by timcanham** [comment]
We don't support USB yet since it isn't typical for the embedded platforms we usually implement for. If you would like to attempt to write one, you can submit it to the [F Prime community](https://github.com/fprime-community) organization as a repo and others could benefit.
